src/Curl/CaseInsensitiveArray.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Curl;
 
+use ReturnTypeWillChange;
```

Comment:
I think this line can be removed

Reply:
Then `#[\ReturnTypeWillChange]` need to be used instead of `#[ReturnTypeWillChange]`. 

What is preferred?

Reply:
Removed it now